src/components/Footer/Footer.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { FaGithub, FaTwitter, FaLinkedin } from "react-icons/fa";
+import { AiOutlineGlobal } from "react-icons/ai";
+import "./footer.css";
+
+export const Footer = () => {
+  return (
+    <footer>
+      <a href="https://twitter.com/partha_sarma8" target="__blank">
+        <FaTwitter />
```

Comment:
Suggestion keep the whole application in either js or ts

Reply:
arey haaa! I missed this since i copy pasted it from other project. Thank you!

Reply:
arey haaa! I missed this since i copy pasted it from other project. Thank you!